projects/npm-tools/packages/npm-scripts/src/config/babel.json

```diff
@@ -4,7 +4,15 @@
 			"plugins": ["transform-react-remove-prop-types"]
 		}
 	},
-	"presets": ["@babel/preset-env", "@babel/preset-react"],
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"targets": "Chrome 65, Firefox ESR, Safari >= 12, last 1 Chrome version, last 1 ChromeAndroid version, last 1 Edge version, last 1 Firefox version, last 1 iOS version"
```

Comment:
Not super important, but for readability I think we can make this something like 

```js
targets: {
    "chrome": "65",
    "safari": ">= 12",
    // ...
}
```

Reply:
AFAIK, that's not supported. [The docs](https://babeljs.io/docs/en/babel-preset-env) say that when you use key-value format you can supply:

> an object of minimum environment versions to support

and they give this example:

```
{
  "targets": {
    "chrome": "58",
    "ie": "11"
  }
}
```

which they contrast with the alternative (that I've used here) of:

> a browserslist-compatible query:
>
> ```
> {
>   "targets": "> 0.25%, not dead"
> }
> ```

So my reading of that is that the query-syntax can't be used when using object keys, only a concrete minimum version can.

Reply:
Strange. I remember seeing an example of using key-value pairs with similar syntax, I can't seem to find where I saw that at though... I may just be delusional 🤷‍♂️

Reply:
Just to rule this out as a possibility, I tried reforming it as:

```
"targets": {
	"Chrome": "65, last 1 version",
	"ChromeAndroid": "last 1 version",
	"Edge": "last 1 version",
	"Firefox ESR": "last 1 version",
	"Firefox": "last 1 version",
	"Safari": ">= 12",
	"iOS": "last iOS version"
}
```

But that gave errors like these:

```
Error: [BABEL] Invalid Option: 'Chrome' is not a valid target
        Maybe you meant to use 'chrome'?
Error: [BABEL] 'ChromeAndroid' is not a valid target
        Maybe you meant to use 'android'?
Error: [BABEL] Invalid Option: 'Edge' is not a valid target
        Maybe you meant to use 'edge'?
Error: [BABEL] Invalid Option: 'Firefox ESR' is not a valid target
        Maybe you meant to use 'firefox'?
Error: [BABEL] Invalid Option: 'Safari' is not a valid target
        Maybe you meant to use 'safari'?
Error: [BABEL] Invalid Option: 'iOS' is not a valid target
        Maybe you meant to use 'ie'?
```

So massaged it a bit further to this (note that for this experiment I dropped "iOS" beacuse I don't know what to put for that):

```
"targets": {
	"chrome": "65, last 1 version",
	"android": "last 1 version",
	"edge": "last 1 version",
	"firefox": "ESR, last 1 version",
	"safari": ">= 12"
}
```

But that yielded:

```
Error: [BABEL] Invalid Option: 'last 1 version' is not a valid value for 'targets.android'.
```

It looks like you can't use the query syntax in this way. My guess is that they are using `parseInt` internally to identify the minimum version, which is why `"65, last 1 version"` works (`65`) but `"last 1 version"` does not (`NaN`).